pennylane/devices/experimental/device_api.py

```diff
@@ -131,7 +131,7 @@ def name(self) -> str:
             self.tracker.record()
     """
 
-    def __init__(self, *args, **kwargs) -> None:
+    def __init__(self) -> None:
```

Comment:
I see that you removed super().__init__ from default_qubit_2 following the pylint warning.
Could you please explain how the tracker is being initialized now?

Reply:
Since `DefaultQubit2` doesn't override the `__init__` method, `Device.__init__` is run, which initializes the tracker.

Reply:
Makes sense. Thank you for clarifying. :slightly_smiling_face: